src/app/core/config/Beispielconfig auf Deutsch.json

```diff
@@ -0,0 +1,1634 @@
+                    }
+                ]
+            },
+            "filters": []
+        }
+    },
+    "view:recurring-activity/:id": {
+        "component": "EntityDetails",
+        "config": {
+            "entity": "RecurringActivity",
+            "panels": [
+                {
+                    "title": "Aktivität",
+                    "components": [
+                        {
+                            "component": "Form",
+                            "config": {
+                                "cols": [
+                                    [
+                                        {
+                                            "input": "text",
+                                            "id": "title",
+                                            "placeholder": "Titel"
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "id": "type",
+                                            "input": "configurable-enum-select",
+                                            "enumId": "interaction-type",
+                                            "placeholder": "Typ"
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "input": "entity-select",
+                                            "id": "assignedTo",
+                                            "entityType": "User",
+                                            "placeholder": "Koordinator:in hinzufügen",
+                                            "label": "Koordinator:innen"
+                                        }
+                                    ]
+                                ]
+                            }
+                        }
+                    ]
+                },
+                {
+                    "title": "Teilnehmer:innen",
+                    "components": [
+                        {
+                            "component": "ActivityParticipantsSection"
+                        }
+                    ]
+                },
+                {
+                    "title": "Events & Anwesenheit",
+                    "components": [
+                        {
+                            "component": "ActivityAttendanceSection"
+                        }
+                    ]
+                }
+            ],
+            "icon": "calendar"
+        }
+    },
+    "view:report": {
+        "component": "Reporting",
+        "config": {
+            "reports": [
+                {
+                    "title": "Basisbericht",
+                    "aggregationDefinitions": [
+                        {
+                            "query": "Child:toArray[*isActive=true]",
+                            "label": "All children",
```

Comment:
```suggestion
                            "label": "Kinder"
```

Reply:
I think the guideline was to rather speak of "Schüler:innen" instead of "children".